a3p-integration/proposals/z:acceptance/test-lib/psm-lib.js

```diff
@@ -64,6 +64,35 @@ const PSM_PAIR = NonNullish(process.env.PSM_PAIR).replace('.', '-');
 const fromBoard = makeFromBoard();
 const marshaller = boardSlottingMarshaller(fromBoard.convertSlotToVal);
 
+/**
+ * @param {string} path
+ */
+const objectFromVstorageEntries = async path => {
+  const rawEntries = await agoric.follow('-lF', `:${path}`, '-o', 'text');
+  return Object.fromEntries(marshaller.fromCapData(JSON.parse(rawEntries)));
+};
+
+const snapshotAgoricNames = async () => {
+  const [brands, instances] = await Promise.all([
+    objectFromVstorageEntries('published.agoricNames.brand'),
+    objectFromVstorageEntries('published.agoricNames.instance'),
+  ]);
+  return { brands, instances };
+};
+
+/**
+ * @param {import('@agoric/ertp').Brand} brand
+ * @param {bigint} numValInPercent
+ */
+const toRatio = (brand, numValInPercent) => {
+  const commonDenominator = AmountMath.make(brand, 10_000n);
```

Comment:
I wonder why the denominator is not `100n`. But I suppose that pre-dates this PR?

Reply:
Yep, I just moved it. :shrug: